packages/guides-restructured-text/src/RestructuredText/Parser/Productions/FieldListRule.php

```diff
@@ -55,8 +55,15 @@ public function apply(BlockContext $blockContext, CompoundNode|null $on = null):
         while ($iterator->valid() && $this->isFieldLine($iterator->current())) {
             $fieldListItemNodes[] = $this->createListItem($blockContext);
             $iterator->next();
+            while ($iterator->valid() && LinesIterator::isEmptyLine($iterator->current())) {
```

Comment:
If you use peek here to get the next line we can remove the prev call... I would rather see prev on the line iterator as a hack. So if we can avoid usage we should do this. 

As far as I know there is right now 1 location using prev, which is on my list to refactor.